src/cryptonote_core/blockchain.cpp

```diff
@@ -97,8 +97,8 @@ static const struct {
   // version 4 starts from block 1220516, which is on or around the 5th of January, 2017. Fork time finalised on 2016-09-18.
   { 4, 1220516, 0, 1483574400 },
   
-  // version 5 starts from block 1406997, which is on or around the 20th of September, 2017. Fork time finalised on 2016-09-18.
-  { 5, 1406997, 0, 1505865600 },  
+  // version 5 starts from block 1288616, which is on or around the 15th of april, 2017. Fork time finalised on 2016-03-14.
+  { 5, 1288616, 0, 1489520158 },  
```

Comment:
You can disregard my IRC post. Six months from the above date, monerod should start notifying about an upcoming fork. That is what the date is about, So, if v6 is still scheduled for 20th of September, that date is about right.